Assets/ConsentManagementProvider/Scripts/interface/ISpSdk.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using System.Collections.Generic;
+
+namespace ConsentManagementProviderLib
+{
+    public interface ISpSdk
+    {
+        void Initialize(
+            int accountId, 
+            int propertyId, 
+            string propertyName,
+            MESSAGE_LANGUAGE language, 
+            string gdprPmId, 
+            string ccpaPmId,
+            string usnatPmId,
```

Comment:
@Nevazhnovu we should remove 
```
            string gdprPmId, 
            string ccpaPmId,
            string usnatPmId,
  ```
  
 since the `LoadPrivacyManager` function already accepts the `pmId` as param

Reply:
Done in https://github.com/SourcePointUSA/unity-sdk/pull/61